src/main/java/org/wso2/securevault/secret/SecretManager.java

```diff
@@ -214,6 +191,51 @@ public void init(Properties properties) {
         initialized = true;
     }
 
+    /**
+     * Check whether to use the provider listed under secretRepositories
```

Comment:
Lets add a fullstop to end of the comment

Reply:
Done